mysql/database.go

```diff
@@ -277,15 +277,13 @@ func (d *database) TableExists(name string) error {
 }
 
 // PrimaryKeys returns the names of all the primary keys on the table.
-func (d *database) PrimaryKeys(tableName string) ([]string, error) {
+func (d *database) PrimaryKeys(tableName string) ([]string, error) {	
 	q := d.Select("k.column_name").
-		From("information_schema.table_constraints AS t").
-		Join("information_schema.key_column_usage AS k").
-		Using("constraint_name", "table_schema", "table_name").
+		Form("information_schema.key_column_usage AS k").
```

Comment:
Looks like you have a typo here. `Form`, it must be `From`.

Reply:
Yes, the second commit fixed bug

https://github.com/upper/db/pull/417/commits/5672173bc9b08a3525394cbecb40e57bd6aba5c5

Reply:
doesn't seem like it